docs/api/public_php_api_search.md

```diff
@@ -244,6 +248,14 @@ You can access the following additional search result data from PagerFanta:
 |[`LocationSearchHitAdapter`](https://github.com/ibexa/core/blob/main/src/lib/Pagination/Pagerfanta/LocationSearchHitAdapter.php)|Makes a Location search against passed Query and  returns [SearchHit](https://github.com/ibexa/core/blob/main/src/contracts/Repository/Values/Content/Search/SearchHit.php) objects instead.|
 |[`ContentFilteringAdapter`](https://github.com/ibexa/core/blob/main/src/lib/Pagination/Pagerfanta/ContentFilteringAdapter.php)|Applies a Content filter and returns a `ContentList` object.|
 |[`LocationFilteringAdapter`](https://github.com/ibexa/core/blob/main/src/lib/Pagination/Pagerfanta/LocationFilteringAdapter.php)|Applies a Location filter and returns a `LocationList` object.|
+|`AttributeDefinitionListAdapter`| Makes a search for product attributes and returns an `AttributeDefinitionListInterface` object. |
+|`AttributeGroupListAdapter`| Makes a search for product attribute groups and returns an `AttributeGroupListInterface` object. |
+|`CurrencyListAdapter`| Makes a search for currencies and returns a list of currencies. |
+|`CustomPricesAdapter`| Makes a search for custom prices and returns a `CustomPrice` object. |
+|`CustomerGroupListAdapter`| Makes a search for customer groups and returns an `CustomerGroupListInterface` object. |
+|`ProductListAdapter`| Makes a search for products and returns an `ProductListInterface` object. |
+|`ProductTypeListAdapter`| Makes a search for product types and returns an `ProductTypeListInterface` object. |
+|`RegionListAdapter`| Makes a search for regions and returns a list of regions. |
```

Comment:
As above, `\Ibexa\Contracts\ProductCatalog\Values\Region\RegionListInterface` is returned here.

```suggestion
|`RegionListAdapter`| Makes a search for regions and returns a `RegionListInterface` object. |
```